paper/automated.tex

```diff
@@ -1,20 +1,108 @@
-\section{Automated Theorem Proving}\label{automated-sec}
+\section{Proof Automation}\label{automated-sec}
 
-\note{TODO: expand this sketch}
+In this project we used proof automation in two ways: automated theorem provers (ATPs) and Lean tactics.
+ATPs are generally stand-alone tools that implement a (semi-) decision procedure for a given formal language or related set of languages.
+For example, Vampire~\cite{DBLP:conf/cav/KovacsV13} is an ATP focused primarily on first-order logic using supperposition, which we used extensively in this project.
+ATPs are generally complex pieces of software, and can often have bugs.
+While we could choose to trust the results of an ATP, and in this project we don't do so either.
```

Comment:
Needs rewrite.

Reply:
I would merge with the previous/next sentence and make it more explicit: "ATPs are complex software that can contain bugs. Instead of trusting ATP output, we used proof certificates, which many ATPs can produce, to reconstruct a proof in Lean.